lookout/style/typos/metrics.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, List, Set, Tuple
+
+import pandas
+from sklearn.metrics import accuracy_score, precision_recall_fscore_support
+
+from lookout.style.typos.utils import Columns
+
+
+def first_k_set(corrections: List[Tuple[str, float]], k: int) -> Set[str]:
+    """
+    Compose the set of k most probable correction candidates (tokens without probabilities).
+
+    :param corrections: List of corrections, sorted by probability.
+    :param k: Number of corrections to take.
+    :return: Set of k most probable correction tokens.
+    """
+    first_k = set()
+    for correction, _prob in corrections[:k]:
+        first_k.add(correction)
+    return first_k
+
+
+def get_scores(data: pandas.DataFrame, suggestions: Dict[int, List[Tuple[str, float]]],
+               mode: str = "correction", k: int = 1) -> Dict[str, float]:
```

Comment:
Idea for a next PR: move these three mode strings to an enumeration, the same way you did with columns